aptos-move/block-executor/src/view.rs

```diff
@@ -240,7 +292,7 @@ impl<'a, T: Transaction, S: TStateView<Key = T::Key>, X: Executable> TResourceVi
     fn get_resource_state_value(
         &self,
         state_key: &Self::Key,
-        _maybe_layout: Option<&Self::Layout>,
+        maybe_layout: Option<&Self::Layout>,
```

Comment:
We used MoveTypeLayout in other functions in the file. Is Self::Layout different than MoveTypeLayout?

Reply:
Self::Layout is an associated type for the trait (https://doc.rust-lang.org/rust-by-example/generics/assoc_items/types.html), so it is actually the same as MoveTypeLayout for this implementation